src/dmd/json.d

```diff
@@ -839,18 +902,50 @@ public:
     private void generateBuildInfo()
     {
         objectStart();
-        property("cwd", getcwd(null, 0));
-        property("config", global.inifilename ? global.inifilename : null);
-        if (global.params.lib) {
-            property("library", global.params.libname);
-        }
+        propertyOrNull("cwd", getcwd(null, 0));
+        propertyOrNull("argv0", global.params.argv0);
+        propertyOrNull("config", global.inifilename);
+        propertyOrNull("libname", global.params.libname);
```

Comment:
Everything else uses camelCase, might be a good idea to use `libName` too?
Precedent is above: we don't use `inifilename`, but `config`